frontend/src/app/ui/components/modals/export-midi-modal/export-midi-modal.component.html

```
@@ -0,0 +1,44 @@
+              @if (fileName.errors?.['pattern']) {
+                <small>{{ 'exportMidi.validation.pattern' | translate }}</small>
+              }
+            </div>
+          }
+        </div>
+
+        <div class="form-group">
+          <label for="midiLoopCount">{{ 'exportMidi.duration' | translate }}</label>
+          <select id="midiLoopCount" formControlName="loopCount" class="form-select">
+            @for (count of loopCounts; track count) {
+              <option [value]="count">{{ count }} {{ count > 1 ? ('export.loops' | translate) : ('export.loop' | translate) }}</option>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)